pkg/cmd/roachtest/spec/option.go

```diff
@@ -33,11 +33,20 @@ func CPU(n int) Option {
 	}
 }
 
-// WorkloadNode indicates that the last node is a workload node.
-// Defaults to a VM with 4 CPUs if not specified by WorkloadNodeCPUs.
+// WorkloadNodeCount indicates the count of last nodes in cluster to be treated
+// as workload node. Defaults to a VM with 4 CPUs if not specified by
+// WorkloadNodeCPUs.
+func WorkloadNodeCount(n int) Option {
+	return func(spec *ClusterSpec) {
+		spec.WorkloadNodeCount = n
+		spec.WorkloadNode = true
+	}
+}
+
 func WorkloadNode() Option {
```

Comment:
nit: My first intuition was that you need to specify both a `WorkloadNode` followed by a `WorkloadNodeCount`. Judging from the TODO above, you plan on replacing `WorkloadNode` which is great, but maybe add that TODO here as well, since here is the comment that would pop up when people hover over the option.